pkgs/applications/virtualization/podman/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchFromGitHub, removeReferencesTo, pkgconfig
+, go, gpgme, lvm2, btrfs-progs, libseccomp
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "podman-${version}";
+  version = "0.11.1.1";
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "libpod";
+    rev = "v${version}";
+    sha256 = "18r7jasaf18cbraf5v2fl96hs47d3ivjq82pivw9knbwafsscg64";
+  };
+  
+  # Optimizations break compilation of libseccomp c bindings
+  hardeningDisable = [ "fortify" ];
+  nativeBuildInputs = [ pkgconfig ];
+  
+  buildInputs = [
+    removeReferencesTo go btrfs-progs libseccomp gpgme lvm2
+  ];
+
+  buildPhase = ''
+    patchShebangs .
+    mkdir -p .gopath/src/github.com/containers
+    ln -sf $PWD .gopath/src/github.com/containers/libpod
+    ln -sf $PWD/vendor/github.com/varlink .gopath/src/github.com/varlink
+    export GOPATH="$PWD/.gopath:$GOPATH"
+    make binaries
+  '';
+  
+  installPhase = ''
+    install -Dm755 bin/podman $out/bin/podman
+  '';
+  
+  preFixup = ''
+    find $out -type f -exec remove-references-to -t ${go} -t ${stdenv.cc.cc} '{}' +
+    find $out -type f -exec remove-references-to -t ${stdenv.glibc.dev} '{}' +
```

Comment:
Can't you combine this in a single command? You can put ` \` at the end of line to break lines in bash if they are too long.